R/serialized_tests.R

```diff
@@ -220,8 +223,9 @@ stop_insufficient_r_version <- function() {
 #' out manually.
 generate_test_samples <- function() {
   gen <- function(x) {
-    if (length(x) == 0) ""
-    else {
+    if (length(x) == 0) {
```

Comment:
Did we have a helper function for this? (Not relevant to this PR.)

Reply:
I just c/p your code from ##279.

Reply:
So I suppose no.

Reply:
Oh, this is just the generator. Got it.